.github/workflows/main.yml

```
@@ -1,5 +1,5 @@
 name: Build Extension
-on: [push, pull_request]
+on: pull_request
```

Suggest we still run on pushes, only for the default branch. That way we have a constant indicator of whether the build is passing on the codebase.
```suggestion
on:
  pull_request:
  push:
    branches:
      - main
```

---

Sure.